### PR TITLE
Add zsh completion

### PR DESCRIPTION
**Pull request by ahmubashshir**
See-also: #173

Signed-off-by: Mubashshir <ahmubashshir@gmail.com>

### CODE REVIEW

**Review by iamkroot — CHANGES_REQUESTED** [review]
Thanks for raising this PR!
While the core completions are fine, I think we should also add instructions on how to actually install them, and also how to generate them for the future.
Maybe add a README file in the same directory?

File: completions/trakts.zsh [hunk 1]
```diff
@@ -0,0 +1,297 @@
+#compdef trakts
+#
+# Zsh auto complete function for trakts
+#
+# shellcheck shell=bash
```
Comment (iamkroot) [inline completions/trakts.zsh:5]: Nitpick: Shouldn't this be zsh?

**Comment by ahmubashshir** [comment]
shellcheck only supports sh/bash/ksh, and bash is the most similar to 
zsh (that's why I disabled shellcheck warnings on zsh specific blocks)
	Mubashshir

File: completions/trakts.zsh (hunk 1 above)
Comment (ahmubashshir) [reply completions/trakts.zsh:5]: shellcheck only supports sh/bash/ksh, and bash is the most similar to
zsh (that's why I disabled shellcheck warnings on zsh specific blocks)

**Comment by ahmubashshir** [comment]
BTW, we can submit the completion to [zsh-completions](https://github.com/zsh-users/zsh-completions) too.

**Review by iamkroot — APPROVED** [review]

**Comment by iamkroot** [comment]
Thanks for the effort @ahmubashshir! :)

**Pull request merged**